execute/executor_test.go

```diff
@@ -687,6 +688,49 @@ func TestExecutor_Execute(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: `multiple unnamed yields`,
+			spec: &plantest.PlanSpec{
+				Nodes: []plan.Node{
+					plan.CreatePhysicalNode("from-test", executetest.NewFromProcedureSpec(
+						[]*executetest.Table{&executetest.Table{
+							KeyCols: []string{"_start", "_stop"},
+							ColMeta: []flux.ColMeta{
+								{Label: "_start", Type: flux.TTime},
+								{Label: "_stop", Type: flux.TTime},
+								{Label: "_time", Type: flux.TTime},
+								{Label: "_value", Type: flux.TFloat},
+							},
+							Data: [][]interface{}{
+								{execute.Time(0), execute.Time(5), execute.Time(0), 1.0},
+								{execute.Time(0), execute.Time(5), execute.Time(1), 2.0},
+								{execute.Time(0), execute.Time(5), execute.Time(2), 3.0},
+								{execute.Time(0), execute.Time(5), execute.Time(3), 4.0},
+								{execute.Time(0), execute.Time(5), execute.Time(4), 5.0},
+							},
+						}},
+					)),
+					plan.CreatePhysicalNode("sum", &universe.SumProcedureSpec{
+						SimpleAggregateConfig: execute.DefaultSimpleAggregateConfig,
+					}),
+					plan.CreatePhysicalNode("yield0", executetest.NewYieldProcedureSpec("_result")),
+					plan.CreatePhysicalNode("yield1", executetest.NewYieldProcedureSpec("_result")),
+				},
+				Edges: [][2]int{
+					{0, 1},
+					{1, 2},
+					{2, 3},
+				},
+			},
+			// XXX(onelson): unsure if the wrapped (inner) error is ever made
+			// visible to the user.
+			// Seems like in this case we'd want the user to get this feedback...
+			wantErr: errors.Wrap(
+				errors.New(codes.Invalid, "tried to produce more than one result with the name \"_result\""),
+				codes.Inherit,
+				"failed to initialize execute state",
+			),
```

Comment:
I'm pretty sure that it is, I've seen lots of errors like this.

Reply:
Okay great. If it comes to it, I think we could add special handling nearer to the top to try and unwrap the inner for certain cases like this. The chain of custody in having this wrapped for now is probably healthy.